test/functionalTest/cases/1048_subscription_cache/three_subscriptions.test

```diff
@@ -337,9 +333,6 @@ Date: REGEX(.*)
         "measuring_interval_in_secs": "REGEX(\d+)", 
         "requestSemaphoreWaitingTime": "REGEX(\d+.\d+)", 
         "statisticsRequests": "3", 
-        "subCacheEntries": "REGEX((1|2))", 
-        "subCacheMutexWaitingTime": "REGEX(\d+.\d+)", 
-        "subCacheRemovals": "1", 
```

Comment:
Yes, I already have these counters back, coming in my next PR, with specific tests